### PR TITLE
Improve run command

### PR DESCRIPTION
**Pull request by viktorprogger**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | ❌

Two enhancements:
- Processing messages from different queue channels
- Ability to set a maximum of jobs to be processed

### CODE REVIEW

File: src/Command/RunCommand.php [hunk 1]
```diff
@@ -7,39 +7,55 @@
...
-        );
+            InputArgument::OPTIONAL | InputArgument::IS_ARRAY,
+            'Queue channel name list to connect to',
+            $this->channels,
+        )
+            ->addOption(
+                'maximum',
+                'm',
+                InputOption::VALUE_REQUIRED,
+                'Maximum number of messages to process in each channel',
```
Comment (xepozz) [inline src/Command/RunCommand.php:44]:
```suggestion
                'Maximum number of messages to process in each channel. Use "0" to process without limits.',
```

File: src/Command/RunCommand.php [hunk 2]
```diff
@@ -7,39 +7,55 @@
...
 
     public function configure(): void
     {
         $this->addArgument(
             'channel',
-            InputArgument::OPTIONAL,
-            'Queue channel name to connect to',
-            QueueFactory::DEFAULT_CHANNEL_NAME
-        );
+            InputArgument::OPTIONAL | InputArgument::IS_ARRAY,
```
Comment (vjik) [inline src/Command/RunCommand.php:36]: Channel is required options.

**Review by vjik — CHANGES_REQUESTED** [review]

File: src/Command/RunCommand.php (hunk 2 above)
Comment (viktorprogger) [reply src/Command/RunCommand.php:36]: Not anymore, see another changes in this PR.

File: src/Command/RunCommand.php (hunk 1 above)
Comment (viktorprogger) [reply src/Command/RunCommand.php]: Thanks, done with another wording

File: config/params.php [hunk 3]
```diff
@@ -19,7 +20,9 @@
     ],
     'yiisoft/yii-queue' => [
         'handlers' => [],
-        'channel-definitions' => [],
+        'channel-definitions' => [
+            QueueFactoryInterface::DEFAULT_CHANNEL_NAME => AdapterInterface::class,
```
Comment (vjik) [inline config/params.php:24]: Should we fix readme [here](https://github.com/yiisoft/yii-queue#ready-for-yiisoftconfig)? Now `AdapterInterface` is optional.

Comment (viktorprogger) [reply config/params.php:24]: It's already fixed in `master`

**Pull request merged**